docs/ntppool/zh_CN/join.html

```diff
@@ -0,0 +1,75 @@
+[% page.title = '加入NTP服务器群！'  %]
+
+<div class="block">
+	<h3><a name="join">我该如何加入pool.ntp.org？</a></h3>
+	<p>
+	首先，我们欢迎你的加入。随着时间服务器群的用量的增加，为了减轻各个服务器的负载，
+	我们能做到的便是增加群内服务器的数量。
+	</p>
+
+	<p>
+	你的电脑<b>必须拥有一个静态的IP地址</b>，并且要有持续不断的，稳定的网络连接。
+	你的IP地址不变或几乎不变对我们来说很重要（比方说，一年换一次IP）。我们对你的网速要求很低。
+	专业一点说，每个用户每几分钟或十几分钟只会与你交换数个UDP数据包。
+	</p>
+
+	<p>
+	目前多数的服务器每秒会收到大概5-15个NTP数据包，高峰期则会增长到每秒大概60-120个NTP数据包。
+	折合成网速大概是1-2KB/s （10-15Kbit/s），高峰期大概8-20KB/s （50-120Kbit/s）。
+	鉴于我们的服务器数量一直在稳步增长，每个服务器的负载理应不会大幅增长。
+	换句话说，你大概有384-512Kbit的上下行网速就够用了。
+	</p>
+	<p>
+	点击数字查看随机一个服务器的负载图表：
```

Comment:
Hmm, the point of the graphs here is to show the load, not something like "hey you can click the numbers too see"...